packages/ui/stories/example/Page.stories.tsx

```diff
@@ -21,6 +21,7 @@ export const LoggedIn = Template.bind({});
 // More on interaction testing: https://storybook.js.org/docs/react/writing-tests/interaction-testing
 LoggedIn.play = async ({ canvasElement }) => {
   const canvas = within(canvasElement);
-  const loginButton = await canvas.getByRole('button', { name: /Log in/i });
+  const loginButton = canvas.getByRole('button', { name: /Log in/i });
+  // eslint-disable-next-line @typescript-eslint/await-thenable
```

Comment:
Is there a way to prevent this issue on other code as well?

Reply:
Can remove the `// eslint-disable-next-line @typescript-eslint/await-thenable` by replacing line 26, `await userEvent.click(loginButton);` with this line of code:

`await Promise.resolve(userEvent.click(loginButton));`

Reply:
Fixed in latest commit

Reply:
FYI we can't wrap this in a promise, because storybook debugger requires use to await directly: https://storybook.js.org/docs/react/essentials/interactions#writing-interactions